.github/workflows/test_linux_packages.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test Linux Packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - ADHOCBUILD
```

Comment:
Leftofter? Or should this rather be an input passed to `workflow_dispatch` and `workflow_call`, similar as in the `build_linux_packages` workflow?